src/applications/form-renderer/tests/unit/containers/App.unit.spec.js

```diff
@@ -51,7 +51,8 @@ describe('<App /> component', () => {
 
 describe('<App /> integration', () => {
   const rootReducer = combineReducers({
-    formLoad: formLoadReducer.formLoad,
+    formLoad: reducer.formLoad,
+    ombInfo: reducer.ombInfo,
```

Comment:
This feels odd to me. See more general discussion.

TL;DR it seems like loading OMB info should be _part of_ loading the other form data.